### PR TITLE
Add call a spirit option

### PR DESCRIPTION
**Pull request by isavirag**
Please review this, @casey-collab

### CODE REVIEW

File: english/Booyah/booyah.md [hunk 1]
```diff
@@ -4,3 +4,4 @@ What am I going to do with my new found knowledge?
 The world is now my [oyster.](https://en.wikipedia.org/wiki/Pacific_oyster)
 
 Have some booyah stew
+or call a [spirit](call-spirit/call-spirit.md)
```
Comment (casey-collab) [inline english/Booyah/booyah.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/call-spirit/call-spirit.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The spirit says everything in your life is upside down! Are you even alive? Is everyone who they say the are?
```
Comment (casey-collab) [inline english/call-spirit/call-spirit.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by isavirag** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :night_with_stars:

**Pull request merged**